test/testground/compositions/utils.go

```diff
@@ -85,7 +95,7 @@ func parseBandwidth(s string) (uint64, error) {
 		return 0, fmt.Errorf("unknown unit in string: %s", s)
 	}
 
-	numberStr := strings.TrimRight(s, "KibMibGibTibKBMGBT")
+	numberStr := strings.TrimRight(s, "KMGTib")
```

Comment:
the only reason this was shortened was because of the linter. I can see how this isn't the most clear, but for now since this is just a testground helper, provided its working as intented I'm just going to leave it

Reply:
if more than one reviewer thinks this is a good idea I'm happy to change

Reply:
Not a big deal, can ship as is 🚢